mission/task_manager/scripts/task_manager_server_node.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python3
+import rospy
+
+from dynamic_reconfigure.server import Server
+from task_manager.cfg import BelugaFSMConfig
+
+from task_manager_defines import defines
+
+import test_1_node
+import test_2_node
+
+
+def callback(config, level):
+    rospy.loginfo("""State change request: {Tac_states}""".format(**config))
+
+    if config["Tac_states"] == defines.Tasks.test_1.id:
+        # Stop all other nodes from last state.
+        rospy.set_param("/tasks/task_2", False)
+        rospy.set_param("/tasks/joystick", False)
+        rospy.set_param("/tasks/valve_vertical", False)
+        rospy.set_param("/tasks/valve_horisontal", False)
+        rospy.set_param("/tasks/pipeline_inspection", False)
+        # Start the node here
+        rospy.set_param("/tasks/task_1", True)
```

Comment:
Make a dict or some other container with task names and corresponding state, and make this a function that firstly sets all params to false, then toggles the specific task. For example

```python

tasks = {"name1": False, "name2": False, ...}

def toggle_task_with_name(name: str) -> None:
    for key, value in tasks.items():
        tasks[key] = False

    tasks[name] = True
```

Reply:
done :)